content/python.rst

```diff
@@ -106,6 +106,14 @@ NO
     other_bad = "string"
     some_bad = "string 'b' yes"
 
+* As we look forward to support Python 3, we should use
+  `from __future__ import unicode_literals`, as it helps with readability,
+  instead of explicitly having to mark every text string as `u'unicode'`.
+  More details here:
+  https://python-future.org/compatible_idioms.html#strings-and-bytes
+
+* All byte strings should be explicitly marked as in `b'byte string`.
```

Comment:
This is incorrect: it should be `b'byte string'`. It's missing the final single quote.